### PR TITLE
refactor(forms): change FormUtils to include Placement

### PR DESCRIPTION
**Pull request by krsween**
Placements don't get the entity pickers.

##### **What did you change?**

Added Placements to the TO_ONE and TO_MANY types that get pickers and chips.

##### **Reviewers**
* @jgodi 

##### **Checklist (completed via merger)**
* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [contributing guide](https://github.com/bullhorn/novo-elements/blob/master/CONTRIBUTING.md)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Visually tested in supported browsers and devices

### CODE REVIEW

**Comment by krsween** [comment]
@jgodi hold off on this PR. I want to at least push a TDD stub.

**Comment by krsween** [comment]
Provided these specs are all good. This is stable.

File: src/elements/form/FormUtils.js [hunk 1]
```diff
@@ -91,9 +90,7 @@ export class FormUtils {
         }
 
         // Overrides
-        if (type === 'checkbox' && field.options) {
```
Comment (krsween) [inline src/elements/form/FormUtils.js]: @jgodi There is no setter in this code block that sets `type` equal to 'checkbox', so this check would never be true.

**Comment by krsween** [comment]
Pretty sure **coveralls** is lying as I just added 180 lines of tests where there were previously 0.

**Comment by jgodi** [comment]
But before this file was not reported coverage on since there was not a spec :P

File: src/elements/form/FormUtils.js (hunk 1 above)
Comment (jgodi) [inline src/elements/form/FormUtils.js]: This was from the old-old-old FormUtils before the first major Form Refactor

**Pull request merged**